.markdownlint.json

```diff
@@ -2,4 +2,5 @@
     "MD007": { "indent": 4 },
     "MD013": false,
     "MD046": { "style": "fenced" }
+    "MD036": { "style": "fenced" }
```

Comment:
Can you explain this change?

From what I see `MD036` is for "Emphasis used instead of header", and I'm not sure it takes a fencing option. It may not take any options.